docs/operation.md

```diff
@@ -97,24 +97,24 @@ If the input data is multi-row, then __Diffix for Desktop__ gives you the choice
-| --------------------------------------------------------------------------------------------------------------- | ----------- | -------- | ----------- |
-| `low_thresh` - The lower bound for the noisy suppression threshold                                              | 3           | 4        | 5           |
-| `sd_supp` - The standard deviation of the suppression Gaussian noise                                            | 1.0         | 1.0      | 1.0         |
-| `low_mean_gap` - The number of standard deviations (`sd_supp`) between `low_thresh` and the Gaussian noise mean | 2           | 2        | 2           |
-| `base_sd` - The standard deviation of the noise                                                                 | 1.5         | 2.25     | 3.0         |
-| `outlier_min` - The minimum possible value of `outlier_count`                                                   | 1           | 2        | 2           |
-| `outlier_max` - The maximum possible value of `outlier_count`                                                   | 2           | 3        | 3           |
-| `top_min` - The minimum possible value of `top_count`                                                           | 3           | 3        | 3           |
-| `top_max` - The maximum possible value of `top_count`                                                           | 4           | 4        | 4           |
+| Parameter                                                                                                       | Default     | Configurable? |
+| --------------------------------------------------------------------------------------------------------------- | ----------- | ------------- |
+| `low_thresh` - The lower bound for the noisy suppression threshold                                              | 3           | yes           |
+| `sd_supp` - The standard deviation of the suppression Gaussian noise                                            | 1.0         | no            |
+| `low_mean_gap` - The number of standard deviations (`sd_supp`) between `low_thresh` and the Gaussian noise mean | 2           | no            |
+| `base_sd` - The standard deviation of the noise                                                                 | 1.5         | no            |
+| `outlier_min` - The minimum possible value of `outlier_count`                                                   | 1           | no            |
+| `outlier_max` - The maximum possible value of `outlier_count`                                                   | 2           | no            |
+| `top_min` - The minimum possible value of `top_count`                                                           | 3           | no            |
+| `top_max` - The maximum possible value of `top_count`                                                           | 4           | no            |
```

Comment:
It probably doesn't make sense to mention these parameters anymore. The information is not actionable anyway.

Reply:
I left it in on purpose, given the previous discussions, that we will provide configurability if needed. In this context I think it's good to inform about the broad choice somewhere. So, the action a user might take is to request configurability .Also this serves as documentation of the values to compare other materials (e.g. paper). 

WDYT?

Reply:
This should be documentation for the app, not for the Diffix mechanism. And the app doesn't use these parameters.
As things are, I think this hinders, more than it helps.

Reply:
I'm not sure about this (right now the users who look for the anonymization parameters used and referred to in the papers, won't know that some values were used at all), but I went ahead and removed the table.

I also tweaked a section in the other page, which I think was out-of-date. please take a look 47f1423d

Reply:
Maybe you're right. Paul will go over this part soon and he can add it back.